django/boss/settings/production.py

```diff
@@ -41,19 +41,21 @@
     }
 }
 
-CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": "redis://{}:6379/3".format(config['aws']['cache']),
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
+if config['aws']['cache-session'] != '':
```

Comment:
I guess it will be fine that we don't have a session cache on dev stacks?  We'll have to test it and see what happens?

Reply:
I manually forced my dev stack to use a session cache to test.

Reply:
As long as there is a single instance in the ASG then there is no problem. The redis cache is needed to multiple instances. Initially we only ran one instance and didn't have the redis session configuration.